requirements-all.txt

```diff
@@ -1,4 +1,6 @@
+django-cors-middleware==1.5.0
```

Comment:
Do we still need this (and `corsheaders` Django app) if we're handling everything at nginx level?

Reply:
nop. removing it.